Fix for Label nfd.node.kubernetes.io/node-name (#42997)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/_index.md

managed by [Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/) (NFD).
NFD uses this for an internal mechanism. You should not edit this annotation yourself.

### nfd.node.kubernetes.io/node-name

Type: Annotation

Example: `nfd.node.kubernetes.io/node-name: node-1`

Used on: Nodes

It specifies which node the NodeFeature object is targeting.

> Creators of NodeFeature objects must set this label and 

Reviewer: Is this an annotation or a label?

```suggestion
Creators of NodeFeature objects must set this annotation and 
```

> consumers of the objects are supposed to use the label for 

Reviewer: Is this an annotation or a label?
An annotation cannot be used to filter objects.

Reviewer: If it is a label, you will need to change line 1471, correct?
On line 1478, you are also referring this as an annotation too.

Author: > If it is a label, you will need to change line 1471, correct? On line 1478, you are also referring this as an annotation too.

Done the changes

filtering features designated for a certain node.

{{< note >}}

> These annotations only applies to nodes where NFD is running.

Reviewer: If this note is for the above ones, you will say "These labels or annotations".

> These Node Feature Discovery (NFD) annotations only apply to 

Reviewer: ```suggestion
These Node Feature Discovery (NFD) labels or annotations only apply to 
```

managed by [Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/) (NFD).
NFD uses this for an internal mechanism. You should not edit this annotation yourself.

### nfd.node.kubernetes.io/node-name

Type: Annotation

Example: `nfd.node.kubernetes.io/node-name: node-1`

Used on: Nodes

It specifies which node the NodeFeature object is targeting.
Creators of NodeFeature objects must set this label and 
consumers of the objects are supposed to use the label for 
filtering features designated for a certain node.

{{< note >}}

> These annotations only applies to nodes where NFD is running.

Reviewer 2: ```suggestion
This annotation only applies to nodes where NFD is running.
```
Would you mind to change this grammar issue?

Author: @windsonsea 
Note is written for [nfd.node.kubernetes.io/**] annotations.  
how about below content

> Node Feature Discovery(NFD) annotations only applies to the nodes where NFD is running.

Reviewer 2: Main concern is that `annotations` does not match `applies`.
If we add NFD, it may look like:
```
These Node Feature Discovery (NFD) annotations only apply to the nodes where NFD is running.
```

Author: Understood, i will make the proposed changes.

Author: As far as i know
1. it is a label
2. [Note] for above 5 nfd.node.kubernetes.io/*
3. These annotation are often used  for NFD
Example:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/02dd550d3bacbae6b24ded0079d00caba09b5c4b/docs/usage/customization-guide.md?plain=1#L58
